bitcoin-units/Cargo.toml

```diff
@@ -0,0 +1,36 @@
+[package]
+name = "bitcoin-units"
+version = "0.1.0"
+authors = ["Andrew Poelstra <apoelstra@wpsoftware.net>", "The Rust Bitcoin developers"]
+license = "CC0-1.0"
+homepage = "https://github.com/rust-bitcoin/rust-bitcoin/"
+repository = "https://github.com/rust-bitcoin/rust-bitcoin/"
+documentation = "https://docs.rs/bitcoin/"
+description = "Bitcoin units"
+keywords = ["crypto", "bitcoin"]
+readme = "README.md"
+edition = "2018"
+
```

Comment:
[`categories`](https://crates.io/category_slugs) is missing, probably only `cryptography::cryptocurrencies`

Reply:
Done, thanks.